net-im/psi/psi-9999.ebuild

```diff
@@ -88,78 +88,82 @@ pkg_setup() {
 }
 
 src_unpack() {
-	git-2_src_unpack
-	unset EGIT_HAS_SUBMODULES EGIT_NONBARE
+	git-r3_src_unpack
 
 	# fetch translations
-	mkdir "${WORKDIR}/psi-l10n"
-	unset EGIT_MASTER EGIT_BRANCH EGIT_COMMIT
-	EGIT_REPO_URI="${LANGS_REPO_URI}" \
-	EGIT_SOURCEDIR="${WORKDIR}/psi-l10n" git-2_src_unpack
+	unset EGIT_BRANCH EGIT_COMMIT
+	if use extras; then
+		EGIT_REPO_URI="${PSI_PLUS_LANGS_URI}"
+	else
+		EGIT_REPO_URI="${PSI_LANGS_URI}"
+	fi
```

Comment:
tip (you don't have to do this): `EGIT_REPO_URI=$(usex extras "${PSI_PLUS_LANGS_URI}" "${PSI_LANGS_URI}")`